Makd.mak

```diff
@@ -28,6 +28,8 @@ PROJECT_NAME ?= $(shell basename $T)
 
 # Define the valid flavors
 VALID_FLAVORS := devel production
+# Aliases for the above (in the same order)
+VALID_FLAVOR_SHORT := development prod
```

Comment:
Shouldn't those be inverted ? Setting `VALID_FLAVORS` to `development production` and the short to `devel prod` ?

Reply:
Yeah, that makes more sense to me too.

Reply:
I'll fix the PR up soon-ish

Reply:
It would be a breaking change, but I would just support the short terms, what's the use for the long versions?